src/status_im/ui/screens/db.cljs

```diff
@@ -216,6 +218,7 @@
                  ::modal
                  ::was-modal?
                  ::rpc-url
+                 ::tooltips
```

Comment:
@rcullito, I believe you implemented tooltips. Could they be moved to :desktop/desktop?

Reply:
@vkjr it was me. Sure they can! Initially I thought about `:desktop/desktop`, but then it crossed my mind that it can be used on mobile as well. Not sure if this was a correct line of thinking.

Reply:
@flexsurfer, what do you think? Should we leave it for future possibility of reusing in mobile?